### ISSUE TITLE
JSON output format

### ISSUE THREAD

**Issue by lgarron** [TeddyHuang-00/sshping#23]
I wanted to contribute JSON output format, but unfortunately the code seems to make some pretty deep assumptions that it's dealing with tabular text data. This is not insurmountable to refactor, but I thought I'd file an issue before giving it a go.

**Comment by TeddyHuang-00** [comment]
Yeah, sure. I was thinking about it the other day. Refactoring is indeed in need as it is written for the sole purpose of printing to terminal. If you'd like to contribute, please don't hesitate and go ahead! :+1:

**Comment by TeddyHuang-00** [comment]
Added in https://github.com/TeddyHuang-00/sshping/commit/3c9c707b0cecc956900ea6f527070ba64e20eb07

**Comment by lgarron** [comment]
> Added in [3c9c707](https://github.com/TeddyHuang-00/sshping/commit/3c9c707b0cecc956900ea6f527070ba64e20eb07)

ooo thx, glad to see it! 🤩
